demo/python/multiclient-cluster-remote-control/orchestrator/start_orchestrator.py

```diff
@@ -0,0 +1,12 @@
+import securexgboost as xgb
+import os
+
+HOME_DIR = os.path.dirname(os.path.realpath(__file__)) + "/../../../../"
+
+with open("../hosts.config") as f:
+    nodes = f.readlines()
+nodes = [x.strip().split(":")[0] for x in nodes]
+
+enclave = xgb.Enclave(HOME_DIR + "build/enclave/xgboost_enclave.signed")
+print("Waiting for client...")
+xgb.serve(enclave, all_users=["user1", "user2"], nodes=nodes)
```

Comment:
This script starts the orchestrator, while `enclave_serve.py` starts a node in the cluster. Right now they're the same, but I was planning to rewrite this script in a later PR as starting the orchestrator shouldn't need a reference to the enclave, but does right now the way the codebase is currently set up.

Reply:
Oh also, the `start_orchestrator.py` script takes in the IP addresses of the nodes in the cluster and passes it into `serve()` while the `enclave_serve` script doesn't

Reply:
I see. Is it possible to merge this script and enclave_serve into a single script? That would be ideal I suppose, since the orchestrator == the server.

Reply:
The orchestrator script and the server script are a bit different, in that the former takes in node IPs while the latter doesn't. I think having them separate in the demo would also ameliorate any confusion a user would help to distinguish the orchestrator from a node in the enclave cluster.

It's probably also better to keep them separate as the orchestrator script won't require a enclave reference once we refactor the codebase a bit.

Reply:
Cool.